{"version":3,"file":"CustomError.js","sourceRoot":"","sources":["../../src/exception/CustomError.ts"],"names":[],"mappings":""}